### PR TITLE
Template Submission: Astroid Template

### PR DESCRIPTION
**Pull request by semnaveen**

### CODE REVIEW

File: content/joomla/joomdev-astroid.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Astroid Framework"
+details: https://astroidframework.com/
+demo: https://demo.joomdev.com/astroid-framework?utm_source=astroid_framework_home
```
Comment (C-Lodder) [inline content/joomla/joomdev-astroid.md]: Should be `https://demo.joomdev.com/astroid-framework/toolbar` which is the URL to the actual template. The URL provided links to a demo site with the template inside an iframe

File: content/joomla/joomdev-jd-agency.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: "JD Agency"
+details: https://www.joomdev.com/products/templates/jd-agency-template
+demo: http://demo.joomdev.com/agency
```
Comment (C-Lodder) [inline content/joomla/joomdev-jd-agency.md]: same comment as above.

File: content/joomla/joomdev-astroid.md (hunk 1 above)
Comment (semnaveen) [reply content/joomla/joomdev-astroid.md]: I have changed the URL and committed new templates as well. 

Please check if now it is good or not.

File: content/joomla/joomdev-jd-agency.md (hunk 2 above)
Comment (semnaveen) [reply content/joomla/joomdev-jd-agency.md]: Corrected. Please check now.

**Comment by dgrammatiko** [comment]
@semnaveen could you please rebase this PR. Sorry my bad!

**Pull request closed without merging**